gamemode/cl_hud.lua

```diff
@@ -745,7 +746,7 @@ function DR:DrawPlayerHUDSass( x, y )
 	surface.SetDrawColor( HexColor("#909090", alpha/2) )
 	surface.DrawRect( x + 8 + 48, y + h/2 + 8 + 2, 228-16-48-2, 6)
 
-	local maxvel = 1000 -- yeah fuck yall
+	local maxvel = 1500 -- yeah fuck yall
```

Comment:
No, that change comes from PR #61 and is just in the diff because Hainaa's fork is out of date and probably just overwritten the files instead of updating the fork with a merge.